tests/input_files/f3_cha_xwall_6x8x6_grad_ol2p_bs_ramp_ale_mfs.dat

```diff
@@ -83,22 +83,10 @@ NAME                            Fluid_Solver
 SOLVER                          UMFPACK
 ------------------------------------------------------SOLVER 2
 NAME                            AVM3_Solver
-SOLVER                          Belos
-AZSOLVE                         GMRES
-AZPREC                          MueLu
-MUELU_XML_FILE                  xml/multigrid/fluid_template.xml
-AZTOL                           1.0E-4
-AZCONV                          AZ_r0
-AZREUSE                         5
+SOLVER                          UMFPACK
 --------------------------------------------------------------------SOLVER 3
 NAME                            Projection_Solver
-SOLVER                          Belos
-AZSOLVE                         CG
-AZPREC                          ILU
-IFPACKGFILL                     8
-AZTOL                           1.0E-12
-AZCONV                          AZ_r0
-AZREUSE                         5
+SOLVER                          UMFPACK
```

Comment:
SOLVER 2 & 3 are not needed anymore and could be replaced by just using SOLVER 1.